docs/webhooks.md

```diff
@@ -158,6 +159,25 @@ Emitted when user is disconnecting to administration or desk.
 }
 </pre>
 
+
+### `user.availability.updated`
+
+Emitted when user's availability is updated from desk.
+
+<pre class="prettyprint lang-js">{
+    "eventId": "d36cd3c4-2d16-4a77-97c2-620bde859b29",
+    "eventType": "user.availability.updated",
+    "platform": "sd",
+    "projectId": 3030,
```

Comment:
I think @alexduf was right, if we can, we must avoid to indicate real projectId.

```suggestion
    "projectId": 1,
```